websites/T/taiga Bot/presence.ts

```diff
@@ -1,12 +1,17 @@
 const presence = new Presence({
   clientId: "682593223948238849"
 });
-const presenceCommunity = new Presence({
-  clientId: "682593903656173569"
-});
-const presenceAnigifs = new Presence({
-  clientId: "682594082274410511"
-});
+
+function getRow(row: number) {
+  let metas = document.getElementsByTagName("meta");
+  for (let i = 0; i < metas.length; i++) {
+    if (metas[i].getAttribute("property") === "premid:row" + row) {
+      let content = metas[i].getAttribute("content")
+      return content === "" ? undefined : content;
+    }
+  }
+  return "";
+}
```

Comment:
make sure that when one return "" it doesnt actually send data that has empty, just dont send data at all then.